refinery/data_set_manager/search_indexes.py

```diff
@@ -151,8 +152,11 @@ def prepare(self, node):
             if type(value) is set:
                 data[key] = " + ".join(str(i) for i in sorted(value))
 
+        datafile = "" if file_store_item is None else file_store_item.datafile
```

Comment:
Solr is expecting strings for index preparation,  but `FileStoreItem.datafile` returns `FieldFile` instances. 

I believe you would want: `datafile = file_store_item.datafile.name` here instead